app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt

```diff
@@ -157,14 +162,18 @@ private fun DeviceItemTexts(device: Device, placeholder: Boolean, isDebug: Boole
         }
     }
 
-    Text(
-        style = MaterialTheme.wireTypography.body02,
-        color = MaterialTheme.wireColorScheme.onBackground,
-        text = device.name,
-        modifier = Modifier
-            .fillMaxWidth()
-            .shimmerPlaceholder(visible = placeholder)
-    )
+    Row(Modifier.fillMaxWidth()) {
+        Text(
+            style = MaterialTheme.wireTypography.body02,
+            color = MaterialTheme.wireColorScheme.onBackground,
+            text = device.name,
+            modifier = Modifier
+                .wrapContentWidth()
+                .shimmerPlaceholder(visible = placeholder)
+        )
+        Spacer(modifier = Modifier.width(MaterialTheme.wireDimensions.spacing8x))
+        VerifyLabel(device.isVerified, Modifier.wrapContentWidth())
```

Comment:
The same item is used when encountering too many clients after login and in that case, we do not need to display the VerifyLabel